rtloader/demo/main.c

```diff
@@ -78,15 +80,15 @@ int main(int argc, char *argv[])
     char *init_error = NULL;
     // Embed Python2
     if (strcmp(argv[1], "2") == 0) {
-        rtloader = make2(python_home, &init_error);
+        rtloader = make2(python_home, "python2", &init_error);
```

Comment:
It can be but the demo doesn't use this value at all so I used this just as the placeholder. I'll just swap it with an empty string to not cause confusion.